README.rst

```diff
@@ -31,7 +31,16 @@ Installation
         'django_dramatiq_pg',
     ]
 
-3. Create a Registry, and register your tasks
+3. Add database router in settings.py if you support multiple databases
+
+   .. code-block:: python
+
+    DATABASE_ROUTERS = [
+        ...
+        'django_dramatiq_pg.router.Router',
+    ]
```

Comment:
Why add this complexity of the router?

Surely the migration itself can check `schema_editor.connection` ?

Reply:
Without the router, the migration will be applied to the default alias In the case of multiple database support.

So the router is required in case the DB alias is not "default".